blocks/hero/search/home.js

```diff
@@ -81,7 +81,11 @@ async function buildForm() {
         </div>
       </div>
       <button class="filter" type="button" aria-label="More Filters" aria-haspopup="true">
-        <span class="icon icon-filter"></span>
+        <span class="icon icon-filter">
```

Comment:
You have to be careful with this - the `icon` keyword in a span will auto-fill the children with a reference to an SVG icon. if you want to use a literal, you need to remove/change these class names.

or replcae the `filter.svg` icon file with the contents of the added sprite, and remove all children nodes here.

see the `decorateIcons` function [link](https://github.com/hlxsites/hsf-commonmoves/blob/8b2291854c419035bc92f7d15c6fe3afaf3bf8d4/scripts/aem.js#L401-L410)